Python/Tests/Analysis/AstAnalysisTests.cs

```diff
@@ -749,5 +749,52 @@ class BankAccount(object):
             }
         }
         #endregion
+
+        #region Type Shed tests
+
+        private static PythonVersion VersionWithTypeShed =>
+            PythonPaths.Versions.LastOrDefault(v => Directory.Exists(Path.Combine(v.PrefixPath, "Lib", "site-packages", "typeshed")));
+
+        [TestMethod, Priority(0)]
+        public void TypeShedElementTree() {
```

Comment:
I hacked this test to run under a version of python that doesn't have typeshed installed, and it still passes. I'm not sure what to make of that.

Reply:
:( It means our analysis is so good that typeshed doesn't actually help here. Which version did you run it on? The one I tried seemed to need it

Reply:
3.6

Reply:
Also, an easy way to disable type shed is to go into AstPythonInterpreter, search for `IncludeTypeShed` and set it to false (which I'm currently turning into one of our secret registry key settings)

Reply:
In the latest code, you can disable it by setting: `HKEY_CURRENT_USER\Software\Microsoft\PythonTools\15.0\Analysis\Project@UseTypeStubPackages` to `0` (defaults to `1`). You can also set `UseTypeStubPackagesExclusively` to `1` to not merge with the original results.